src/components/playback/playbackmanager.js

```diff
@@ -1848,7 +1848,33 @@ class PlaybackManager {
                     SortBy: options.shuffle ? 'Random' : 'SortName',
                     MediaTypes: 'Audio'
                 });
-            } else if (firstItem.IsFolder && firstItem.CollectionType === 'homevideos') {
+            } else if (firstItem.Type === 'Series'){
+                promise = new Promise(function (resolve, reject) {
+                    const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
+                        apiClient.getEpisodes(firstItem.Id, {
+                            IsVirtualUnaired: false,
+                            IsMissing: false,
+                            UserId: apiClient.getCurrentUserId(),
+                            Fields: 'Chapters'
+                        }).then(function (episodesResult) {
+                            let foundItem = false;
+                            episodesResult.Items = episodesResult.Items.filter(function (e) {
+                                if (foundItem) {
+                                    return true;
+                                }
+                                if (!e.UserData.Played) {
+                                    foundItem = true;
+                                    return true;
+                                }
+
+                                return false;
+                            });
+                            episodesResult.TotalRecordCount = episodesResult.Items.length;
+                            resolve(episodesResult);
+                        }, reject);
+                });
+
```

Comment:
I know you took this from the `Episode` block, but could you make it without using Promise nesting?